de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/spi/impl/OrderLineReceiptScheduleProducer.java

```diff
@@ -91,6 +92,10 @@ private I_M_ReceiptSchedule createReceiptScheduleFromOrderLine(final org.compier
 			receiptSchedule.setRecord_ID(line.getC_OrderLine_ID());
 			receiptSchedule.setC_Order_ID(line.getC_Order_ID());
 			receiptSchedule.setC_OrderLine_ID(line.getC_OrderLine_ID());
+			
+			// FRESH-667
```

Comment:
it's good to add the task#, but please rather add the github issue number, e.g. gh #338 
because external contributors don't know what to do with a FRESH-123 number